src/tridactyl.d.ts

```diff
@@ -16,6 +16,11 @@ interface Window {
     eval(str: string): any
 }
 
+// Firefox Document interface
```

Comment:
We should probably strive to flesh this out and maybe upstream it to web-ext-types?